requirements.txt

```diff
@@ -1,7 +1,7 @@
 # TODO: Remove when pr https://github.com/go-macaroon-bakery/py-macaroon-bakery/pull/95 gets merged
 macaroonbakery==1.3.2
```

Comment:
out-of-scope: Apparently the PR mentioned in the TODO is merged. Should we try to remove this dependency?

Reply:
let's make an issue

Reply:
#195